xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -189,7 +189,7 @@
       <Docs>
         <param name="array">To be added.</param>
```

Comment:
What about these?

Reply:
Yep. Let's get everything documented. Only the remarks are optional, everything else is mandatory.
The text that could not be auto ported with the tool, needs to be documented manually.
Can you please also make sure to add any exceptions you find, @pgovind?